build-system/tasks/validator.js

```diff
@@ -30,6 +30,30 @@ async function validator() {
   execOrDie('cd validator && python build.py' + validatorArgs);
 }
 
+/**
+ * Simple wrapper around the Java validator test suite.
+ */
+async function validatorJava() {
+  const commands = ['cd validator/java'];
+  if (argv.clean) {
+    commands.append('bazel clean');
+  }
+
+  execOrDie(
+    commands
+      .concat([
+        'echo travis_fold:start:java_validator_build',
+        'bazel run //:fetchAMPResources',
+        'bazel build //:amphtml_validator_java_proto_lib',
+        'bazel run //:copyValidatorJavaSource',
+        'bazel build //:amphtml_validator_lib',
+        'echo travis_fold:end:java_validator_build',
+        'bazel run //:amphtml_validator_test',
```

Comment:
For maintainability, let's move these commands into a separate `.sh` or `.py` wrapper.

Reply:
Done; split into `build.sh` and `build_and_test.sh`. PTAL